akka-persistence-typed/src/main/scala/akka/persistence/typed/EventSourcedSignal.scala

```diff
@@ -52,14 +53,14 @@ final case class SnapshotFailed(metadata: SnapshotMetadata, failure: Throwable)
   def getSnapshotMetadata(): SnapshotMetadata = metadata
 }
 
-final case class DeleteSnapshotCompleted(target: DeletionTarget) extends EventSourcedSignal {
+final case class DeleteSnapshotsCompleted(target: DeletionTarget) extends RetentionSignal {
```

Comment:
We delete one or more, simpler with this?

Reply:
Yeah, that was my thought with changing this in the signals PR, the signal is the same, it is just what was deleted that is different. There was no usage because there wasn't any logic implemented to actually delete before this PR.